base/dates/parse.jl

```diff
@@ -110,59 +126,58 @@ function Base.parse(::Type{DateTime}, s::AbstractString, df::DateFormat{Symbol("
     th = tm = ts = tms = Int64(0)
 
     nv, i = tryparsenext_base10(s,i,end_pos,1)
-    dy = isnull(nv) ? (@goto error) : get(nv)
+    dy = isnull(nv) ? (@goto error) : nv.value
```

Comment:
Ah yeah it's exported and documented. Longer to type but presumably more likely to survive any modification to representation of `Nullable`s.